grails-app/views/stockMovement/show.gsp

```diff
@@ -555,10 +556,14 @@
                             <warehouse:message code="documents.label" default="Documents"/>
                         </g:link>
                     </li>
-                    <g:if test="${stockMovement?.requisition}">
-                        <li>
+                    <g:if test="${stockMovement.sourceType == RequisitionSourceType.ELECTRONIC}">
+                        <g:set var="commentCount" value="${stockMovement?.requisition?.comments?.size()}" />
+                        <li
+                            data-count="${commentCount < 1000 ? commentCount : '999+' }"
+                            class="${commentCount > 0 ? 'tab-badge' : ''}"
+                        >
                             <g:link controller="stockMovement" action="comments" params="${[ id: stockMovement?.id ]}">
-                                <warehouse:message code="comments.label" default="Comments"/>
+                                <warehouse:message code="comments.label" default="Comments"/>${fragment}
```

Comment:
what is `${fragment}`?

Reply:
I was testing how to extract # from the URL, but it didn't. Thanks for noticing that, it needs to be removed.